MOM_input

```diff
@@ -641,6 +662,19 @@ RESTORE_SALINITY = True         !   [Boolean] default = False
                                 ! drives sea-surface salinity toward specified values.
 
 ! === module MOM_surface_forcing_nuopc ===
+MAX_P_SURF = 0.0                !   [Pa] default = -1.0
+                                ! The maximum surface pressure that can be exerted by the atmosphere and
+                                ! floating sea-ice or ice shelves. This is needed because the FMS coupling
+                                ! structure does not limit the water that can be frozen out of the ocean and the
+                                ! ice-ocean heat fluxes are treated explicitly.  No limit is applied if a
+                                ! negative value is used.
+USE_RIGID_SEA_ICE = True        !   [Boolean] default = False
+                                ! If true, sea-ice is rigid enough to exert a nonhydrostatic pressure that
+                                ! resist vertical motion.
+SEA_ICE_RIGID_MASS = 100.0      !   [kg m-2] default = 1000.0
+                                ! The mass of sea-ice per unit area at which the sea-ice starts to exhibit
+                                ! rigidity
+
```

Comment:
Are these needed for coupled ice-ocean stability (damping gravity waves in ice-covered regions), or can they be deleted (together with `DYNAMIC_SURFACE_PRESSURE` above)? Maybe GFDL need this for stability because they're not coupling OM5 every timestep...?
```suggestion

```